src/unit_tests/analysisd/test_decoder_syscollector.c

```diff
@@ -2100,13 +2194,13 @@ int main()
         cmocka_unit_test_setup_teardown(test_syscollector_dbsync_valid_msg_unknown_operation, test_setup_valid_msg_unknown_operation, test_cleanup),
         cmocka_unit_test_setup_teardown(test_syscollector_dbsync_valid_msg_invalid_field_list, test_setup_valid_msg_invalid_field_list, test_cleanup),
         cmocka_unit_test_setup_teardown(test_syscollector_dbsync_valid_msg_query_error, test_setup_valid_msg_query_error, test_cleanup),
-        cmocka_unit_test_setup_teardown(test_syscollector_dbsync_hotfixes_valid_msg_with_separator_character, test_setup_hotfixes_valid_msg_with_separator_character, test_cleanup),
```

Comment:
This setup function still exists and it is not used anymore

Reply:
Not as part of this PR, but these setups are not used either: 
- test_setup_valid_msg_no_data()
- test_setup_valid_msg_data_int()
- test_setup_valid_msg_operation_as_number()